src/plugins/dashboard/public/dashboard_container/component/grid/dashboard_grid_item.tsx

```diff
@@ -91,12 +92,18 @@ export const Item = React.forwardRef<HTMLDivElement, Props>(
       }
     }, [id, dashboardApi, scrollToPanelId, highlightPanelId, ref, blurPanel]);
 
+    const dashboardContainerTop =
+      (document.querySelector('.dashboardContainer') as HTMLDivElement)?.offsetTop || 0;
+
     const focusStyles = blurPanel
       ? css`
           pointer-events: none;
           opacity: 0.25;
         `
-      : undefined;
+      : // TODO: use a variable for global nav offset
+        css`
+          scroll-margin-top: ${dashboardContainerTop + 96 + DASHBOARD_MARGIN_SIZE}px;
```

Comment:
The global nav offset can also change if the user has `xpack.banners.placement: top` in their config file. I wonder if we can use something like `const globalNav = document.querySelector('.header__bars').offsetTop || 0` instead of hard-coding `96` as a variable?

Reply:
Addressed in [152122f](https://github.com/elastic/kibana/pull/182535/commits/152122f80e2eee90383fb13e9a0d6d95bcd0cd9b)